### PR TITLE
aks-preview: Add node config cli commands

### PR DESCRIPTION
**Pull request by bowang-666**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

File: src/aks-preview/azext_aks_preview/custom.py [hunk 1]
```diff
@@ -3002,3 +3021,64 @@ def get_aks_custom_headers(aks_custom_headers=None):
...
+    os_config = get_file_json(file_path)
+    if os_config is None:
+        raise CLIError("Error reading Linux OS configuration at {}".format(file_path))
+    config_object = LinuxOSConfig()
+    config_object.transparent_huge_page_enabled = os_config.get("transparentHugePageEnabled", None)
+    config_object.transparent_huge_page_defrag = os_config.get("transparentHugePageDefrag", None)
+    config_object.swap_file_size_mb = os_config.get("swapFileSizeMB", None)
+    # sysctl settings
+    sysctls = os_config.get("sysctls", None)
+    if sysctls is not None:
```
Comment (arrownj) [inline src/aks-preview/azext_aks_preview/custom.py:3053]: How about use `if sysctls:` to cover other empty case (empty string, dict, array, etc) ?

**Comment by yonzhan** [comment]
aks-preview

File: src/aks-preview/azext_aks_preview/custom.py (hunk 1 above)
Comment (bowang-666) [reply src/aks-preview/azext_aks_preview/custom.py]: updated to use isinstance, this actually has to be a dict otherwise get would throw.

**Review by arrownj — APPROVED** [review]
LGTM

**Pull request merged**